### ISSUE TITLE
Exception when using FitBayes in pyimagej

### ISSUE THREAD

**Issue by drmatthews** [flimlib/flimj-ops#17]
When I run the following in a script using pyimagej:

`fitResults = ij.op().run("flim.fitBayes", fitParams)`

I get the following exception:

`java.lang.UnsatisfiedLinkError: java.lang.UnsatisfiedLinkError: 'void flimlib.FLIMLibJNI.Bayes_set_search_grid(float[], float[])'`

Are there any specific parameters related to `fitBayes` I need to set?

**Comment by drmatthews** [comment]
Additionally, this exception also occurs when running the example Groovy script in Fiji when using `fitBayes`.

**Comment by paulbarber** [comment]
Hi Dan,
I tried on my updated version of Fiji and FLIM-ops on Windows 10, and the Groovy script using `fitBayes` works for me.
ImageJ 1.53c
flimj-ops-2.1.1.jar
flimj-ui-1.1.0.jar
From https://sites.imagej.net/FLIMJ/

Maybe related to this commit in that we had problems with these calls in recent versions:
[https://github.com/flimlib/flimj-ops/commit/1d742860bb99f31365a6b3dd5ad19b1883eb4214](https://github.com/flimlib/flimj-ops/commit/1d742860bb99f31365a6b3dd5ad19b1883eb4214)

But sounds like Java cannot find the native library (.dylib on MAC?) for Bayes_set_search_grid()
or that method is not in the version of dylib that you have.

Anyone know how to force a refresh of the dylibs? A fresh install of Fiji may not do that unless you uninstall the old one completely?

Best,

**Comment by ctrueden** [comment]
@paulbarber Sorry for the late reply!

TL;DR: I fixed FLIMJ on macOS and Linux. Run your Updater to get the latest FLIMLib. :smile:

Here is how I figured out what was wrong:

For Windows, we had the following native libs on the FLIMJ update site in [`jars/win64`](https://sites.imagej.net/FLIMJ/jars/win64/):

  | flimlib-2.1.0-natives-windows_64.jar-20200507174431 | 2020-05-07 17:44 | 140K |  
-- | -- | -- | -- | --
  | flimlib-2.1.1-natives-windows_64.jar-20200907172208 | 2020-09-07 16:22 | 140K |  

But for macOS, in [`jars/macosx`](https://sites.imagej.net/FLIMJ/jars/macosx/), we had only:

  | flimlib-2.0.0-natives-osx_64.jar-20191115115302 | 2019-11-15 11:53 | 107K |  
-- | -- | -- | -- | --

So it looks like an update to the native FLIMLib JAR was done in May 2020 for Windows, but not for macOS. And furthermore, there was not even a `jars/linux64` at all! :disappointed: 

I dropped the latest `flimlib-2.1.1-natives` JAR for macOS and for Linux into the `jars/macosx` and `jars/linux64` folders respectively, then uploaded them to the FLIMJ update site. I also tested on my Linux system and FLIMJ now works, whereas previously it crashed.

**Comment by paulbarber** [comment]
Thank you @ctrueden
